### ISSUE TITLE
What are "industrial payloads" exactly?

### ISSUE THREAD

**Issue by gavanderhoorn** [arminstitute/industrial_payload_manager#1]
The wiki page uses the term "industrial payload" a lot, ~~as does the readme here in the repository~~. It's not clear to me what an "industrial payload" is exactly.

Could that be clarified?

Is it an EEF, the object/objects grasped/handled by EEFs? The combination of those? Something else?

**Comment by johnwason** [comment]
The "industrial_payload_manager" has been developed as part of the [ARM Robotic Assistant for Composites Manufacturing](https://john-wen.com/research/projects/robotic-assistant-for-composites-manufacturing) project. For our purposes, an "industrial payload" is a panel with Aruco tags and target points for the vacuum gripper.

I plan to talk with ROS-I to decide what additional information should be included in the Payload definition. This is currently a starting point based on what we needed for our project.

**Comment by gavanderhoorn** [comment]
> For our purposes, an "industrial payload" is a panel with Aruco tags and target points for the vacuum gripper.

Given this description isn't the current name of the package a bit too generic?

> For our purposes, an "industrial payload" is a panel with Aruco tags and target points for the vacuum gripper.

Is the goal to make this package more generic? The current code seems to make quite a few assumptions about these things.

> I plan to talk with ROS-I to decide what additional information should be included in the Payload definition.

Extrapolating (and speculating a bit) I understand that an "industrial payload" is anything that can be picked up / manipulated by a(n industrial) robot?

**Comment by johnwason** [comment]
Yes, the goal is to make it work for basically anything that is picked up by an industrial robot with visual markers and specified grasp points. Currently it is fairly specific to our project, but the expectation is to make it more generic. If you have specific requirements let me know and I will attempt to integrate them.
